data/block/blockV2.proto

```diff
@@ -28,4 +28,6 @@ message MiniBlockReserved {
 message MiniBlockHeaderReserved {
   ProcessingType ExecutionType = 1;
   MiniBlockState State = 2;
+  int32 IndexOfFirstTxProcessed = 3;
```

Comment:
why int32 instead of uint32?

Reply:
Do you check somewhere in the code for `-1` values? if yes, then it's fine

Reply:
Yes a cross mini block not executed yet (pending) has index of last tx processed -1